src/Mvc/test/Microsoft.AspNetCore.Mvc.IntegrationTests/ValidationIntegrationTests.cs

```diff
@@ -1718,11 +1719,12 @@ public async Task FromBody_JToken_ExcludedFromValidation()
             };
 
             var testContext = ModelBindingTestHelper.GetTestContext(
-                request =>
+                updateRequest: request =>
                 {
                     request.Body = new MemoryStream(Encoding.UTF8.GetBytes("{ message: \"Hello\" }"));
                     request.ContentType = "application/json";
-                });
+                },
+                mvcOptions: options);
```

Comment:
Need to use the same `MvcOptions` instance was an odd fallout of removing `AllowValidatingTopLevelNodes`. The `ParameterBinderExtensions` helpers create `ModelMetadata` with a provider that uses these options. The default `MvcOptions` that `GetTestContext(...)` returns does not configure any binder providers.